include/seqan3/alignment/pairwise/policy/all.hpp

```diff
@@ -60,20 +60,13 @@
  * Function name     | Arguments                              | Return value                         |
  * ----------------- | -------------------------------------- | ------------------------------------ |
  * `compute_cell`    | `cell &&`, `cache &`, `score const &`  | `void`                               |
- * `make_cache`      | `gap_scheme const &`                   | alignment_algorithm_state            |
  *
  * - compute_cell:
  *
  *    This function implements the kernel that computes the score and, if enabled, the traceback direction.
  *    It gets as an input the dereferenced value of the scoring matrix (see above), the current alignment algorithm
  *    cache and the score of comparing two letter of the used alphabet using the passed scoring scheme.
  *
- * - make_cache:
- *
- *     Initialises and returns the cache used in the alignment algorithm. It must return a type that allows structured
- *     bindings, with the first argument being of the same type as the allocator value type used in the matrix policy,
- *     the second argument being the costs for gap opening + gap and the third argument being the costs for a gap.
- *
```

Comment:
How is this related to this PR?

Reply:
It is not. I forgot to split it out into a separate PR.